setup.cfg

```diff
@@ -0,0 +1,12 @@
+[metadata]
+name = LWA_EPIC
+long_description = file: README.md
+long_description_content_type = text/markdown
+Classifiers =
+    Programming Language :: Python :: 3
+    Operating System :: Linux
+
+[options]
+packages = find:
+python_requires = >=3.6
+include_package_data = True
```

Comment:
We should explicitly list our dependencies in an install_requires section. I know there is a whole install document to get people going, but being explicit about our installation requirements will help pip help users get things right.

Reply:
Hmm, I'm not super familiar with how `pip` does installations, but I think we have a bit of a conflict here. Most of our dependencies we ask the user to install using conda (see `config/epic.yml` file). So when we get to the `setup.cfg` file, do we only include the dependancies that pip installs?

Reply:
I don't really see a conflict :thinking: pip/conda are just methods through which to get a dependency, but telling the thing that is actually installing your repo what is required to run your repo will ensure that you can actually run the code right? But there is a difficulty with relying on bifrost which requires an interesting installation procedure. Normally pip would just try to grab things and install them itself, but I wonder if there's a way to just tell pip "Don't try to install pacakges if we don't have them, just fail"

Reply:
yeah, that's the conflict I was talking about. I don't want pip to try to install things if we don't have them and want conda to install them. I'll look into whether there's a way to do that.

Reply:
oh then I see _that_ conflict.